README.md

```diff
@@ -224,13 +222,13 @@ If you have already built the generated pages, then the functional tests can be
 
 `yarn test_functional`
 
-This can be limited to tests under a directory with:
+This can be limited to a single spec using:
 
-`yarn test_functional --path tests/functional/spec/components/`
+`./node_modules/.bin/wdio tests/functional/wdio.conf.js --spec tests/functional/spec/save_sign_out.spec.js`
 
-To run a single test, add `@watch` into the name of any `describe` or `it` function and run:
+To run a single test, add `.only` into the name of any `describe` or `it` function and run:
 
-`yarn test_functional --watch`
+`yarn test_functional`
 
 An example of adding `@watch` looks like this:
 `describe('@watch Skip Conditions', function() {...}` or
```

Comment:
The `@watch` example needs to be reworded into a `.only` example or removed.

The Cypress mention below also needs to be removed.